helix-term/src/keymap.rs

```diff
@@ -47,13 +48,13 @@ macro_rules! keymap {
     };
 
     (@trie
-        { $label:literal $($($key:literal)|+ => $value:tt,)+ }
+        { $label:literal $(sticky=$sticky:literal)? $($($key:literal)|+ => $value:tt,)+ }
     ) => {
-        keymap!({ $label $($($key)|+ => $value,)+ })
+        keymap!({ $label $(sticky=$sticky)? $($($key)|+ => $value,)+ })
```

Comment:
Could we turn this into `keymap!(sticky; { })`?

Reply:
Ah I see, it needs to be per submap. I was thinking we could use a better format there too, maybe

```rust
"g" => (STICKY "Goto" {
  // keys..
})
```